addons/mail/static/src/messaging/entity/device/device.js

```diff
@@ -18,6 +18,14 @@ function DeviceFactory({ Entity }) {
             return entity;
         }
 
+        /**
+         * @override
+         */
+        delete() {
+            window.removeEventListener('resize', this._onResize);
+            super.delete();
+        }
```

Comment:
The purpose of `start` is execute some code after it's being created, because we cannot put it in `constructor`/`init`. Code of `stop` should always happen whenever an entity is deleted, and it doesn't make sense to call `stop` without delete, hence the merge.

Reply:
Having start/stop be outside of constructor/delete might not be needed in our case in practice but it's just as easy to code so I see no reason to lose this possibility for stop, especially when start is still standalone.

We could imagine a feature in the future where we would want to put Odoo in "quiet" mode as to not receive bus notifications... Then it would make sense to stop notification handler without deleting it.

Reply:
That's not the intent of `start`/`stop` to enable/disable reception of notification. Some uses of `start()` consists of adding event listeners like with longpolling bus, but the most important one is about initializing messaging. The reasoning of `start()` was to apply some logic on create just when entities have been created. The `stop` we have introduced before was there to remove cleanly remove listeners without deleting the entities, because we didn't care for that... until this PR for which complete teardown of entities is of major importance: any timer/throttle on any thread must be cleared.

Since this PR now cleanly delete all entities after each test, `stop()` no longer makes sense on its own so this logic now has to move to `delete()`.

Reply:
Ok but then I would argue that if we allow ourselves to have "init" now, we should also allow ourselves to have "after init" and move the code from start to there.

Reply:
There is no reason why start should be called manually, for example with autocreate=True start should be called automatically after create.

Reply:
Certainly, such feature does not exist yet, hence the necessity of manual `start()` for the moment.

Reply:
ok for later, should create technical task

Reply:
task-2265992